book.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225521129-9068cf93-1d74-4663-aa51-2eed93deb091.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages capitalize to make your pull request professional :blush: